apps/cli/src/commands-index.ts

```diff
@@ -23,6 +23,7 @@ import estimateMaxSpendable from "./commands/estimateMaxSpendable";
 import exportAccounts from "./commands/exportAccounts";
 import firmwareRepair from "./commands/firmwareRepair";
 import firmwareUpdate from "./commands/firmwareUpdate";
+import firmwareUpdateAction from "./commands/firmwareUpdateAction";
```

Comment:
Maybe we should all name them: `deviceSDK<Something>` to gather all the new CLI using the device SDK (so `deviceSDKFirmwareUpdate`) as long as the device SDK is not mainly used). WDYT ?

Reply:
Done